docker-compose.full.yml

```diff
@@ -166,7 +166,7 @@ services:
       traefik.enable: "false"
 
   node:
-    image: ${NODE_IMAGE:-node:lts-alpine}
```

Comment:
We're having inconsistencies when installing `node:lts-alpine` because it's always getting the latest one instead of the one we need (currently 14).